frontend/src/metabase/selectors/embed.ts

```diff
@@ -18,7 +19,11 @@ export const getIsEmbeddingSdk = (state: State): boolean => {
   return !!state.embed.isEmbeddingSdk;
 };
 
+/**
+ * TODO: Remove this selector and introduce a function in `frontend/src/metabase/embedding/config.ts` instead.
```

Comment:
I wanted to do this in the same PR, but that would make backporting harder and bloating this PR.